src/modules/emote_menu/components/VirtualizedList.jsx

```diff
@@ -0,0 +1,76 @@
+import React, {useCallback, useEffect, useMemo, useRef, useState} from 'react';
+
+const style = (index, rowHeight) => ({
+  position: 'absolute',
+  top: `${index * rowHeight}px`,
+  height: `${rowHeight}px`,
+  width: '100%',
+});
+
+const headerStyle = (rowHeight) => ({
+  position: 'sticky',
```

Comment:
unless you need a value defined in js, css values could be stored in a css module rather than hardcoding them